client/my-sites/plugins/plugin-details-CTA/index.jsx

```diff
@@ -168,7 +167,7 @@ const PluginDetailsCTA = ( { plugin, isPlaceholder } ) => {
 		return <PluginDetailsCTAPlaceholder />;
 	}
 
-	if ( isPluginInstalledOnsiteWithSubscription && sitePlugin ) {
+	if ( isPluginInstalledOnsite && sitePlugin ) {
```

Comment:
@WBerredo this change interferes with freemium (paid) plugins that show as installed even in case they don't have an active subscription.

In this paid listing you will see that the plugin is installed (the free version) but there is no purchase (see the ellipsis menu) and therefore users that have already the free version cannot purchase the paid.

![CleanShot 2022-11-16 at 12 12 10@2x](https://user-images.githubusercontent.com/12430020/202152857-4cf931ff-1eab-4d3b-ad2f-cc07ba73131b.png)

Is there any reason we wanted this change?

Reply:
We do have another case where a marketplace plugin is active but doesn't have a purchase. That happens when you upload a premium plugin therefore it won't have a purchase associated. 

We need to find a differentiation between the two cases (freemium and premium uploaded) to show the correct CTAs for each.

Reply:
Do you know any property that we could use that flags it is freemium or it was uploaded?

Reply:
> We do have another case where a marketplace plugin is active but doesn't have a purchase. That happens when you upload a premium plugin therefore it won't have a purchase associated.
> 
> We need to find a differentiation between the two cases (freemium and premium uploaded) to show the correct CTAs for each.

Even in those cases, shouldn't the customer be able to buy the subscription from WordPress.com? I can understand though that "inactive" wording doesn't help, it should rather be "subscription inactive"?

Reply:
Currently, we either show the purchase option or we show the menu to manage the plugin installation. In those cases should we show both?

Reply:
> Currently, we either show the purchase option or we show the menu to manage the plugin installation. In those cases should we show both?

Doesn't sound good or bad - maybe it's midterm solution?

Reply:
I implemented this solution here: #70393